index.html

```diff
@@ -15,8 +15,8 @@ <h1 id="logo">Aziz Mejri</h1>
         <i class="fas fa-bars"></i>
       </button>
       <nav id="mobile-nav">
-        <button type="button" class="menu-bars">
-          <i class="fas fa-bars"></i>
+        <button type="button" class="menu-bars x-mark">
+          X
         </button>
         <ul>
```

Comment:
- [ ] Please🙏  add the functionality that when the user clicks or taps any of the mobile menu options, the mobile menu disappears. This provides a better user experience😃 as users expect the menu to disappear when they click on a menu item and make sure it enables them to navigate to the correct page section.